### ISSUE TITLE
[no-call-expression] Add configuration options (for `*ngIf` or some method names)

### ISSUE THREAD

**Issue by GerkinDev** [angular-eslint/angular-eslint#429]
In some cases, call expressions are actually pertinent and should not be reported. For example, when working with Angular Material and reactive forms, it is common to have patterns like this:

```html
<mat-form-field>
	<mat-label>My field</mat-label>
	<input matInput [formControl]="fieldFormControl"/>
	<mat-error *ngIf="fieldFormControl.hasError('foo')">Not a foo<mat-error>
	<mat-error *ngIf="fieldFormControl.hasError('bar')">Not a bar</mat-error>
</mat-form-field>
```
Such pattern creates a lot of warnings, so the main options here are
* being spammed with warns that are actually not pertinent
* disable the rule for the block/line
* disable the rule altogether

It would be far more convinient to allow **some** cases, that might be specified by configuration. Maybe an expression regex (to match `hasError`) or a directive whitelist.

---

It would be nice to have an issue template for feature requests.

**Comment by rafaelss95** [comment]
Hey @GerkinDev, the point of this rule is precisely to avoid method calls in the template to prevent performance problems, because as we know if we use this, it will be called in each CD cycle and, for this reason, I don't think it makes sense to ignore cases like the ones mentioned.

That said, if you allow me to give you an advice, you could create a `@Pipe` for this instead:

```typescript
@Pipe({ name: 'appControlHasError' })
export class ControlHasErrorPipe implements PipeTransform {
  transform(control: AbstractControl, errorCode: string): Observable<boolean> {
    return control.statusChanges.pipe(
      startWith(control.status),
      map(() => control.hasError(errorCode)),
    );
  }
}
```

```html
<mat-error *ngIf="fieldFormControl | appControlHasError: 'foo' | async">Not a foo</mat-error>
<mat-error *ngIf="fieldFormControl | appControlHasError: 'bar' | async">Not a bar</mat-error>
```

[**DEMO**](https://stackblitz.com/edit/angular-3mk5dn)

**Comment by GerkinDev** [comment]
This is actually an awesome idea :) I'll go for it! Thanks!
